The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start

waiting for server to start....2025-02-14 09:02:49.879 UTC [46] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:02:49.886 UTC [47] LOG:  database system was shut down at 2025-02-14 09:02:49 UTC
2025-02-14 09:02:49.889 UTC [46] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down...2025-02-14 09:02:50.251 UTC [46] LOG:  received fast shutdown request
.2025-02-14 09:02:50.252 UTC [46] LOG:  aborting any active transactions
2025-02-14 09:02:50.254 UTC [46] LOG:  background worker "logical replication launcher" (PID 53) exited with exit code 1
2025-02-14 09:02:50.255 UTC [48] LOG:  shutting down
2025-02-14 09:02:50.265 UTC [46] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 09:02:50.358 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 09:02:50.358 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 09:02:50.360 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:02:50.371 UTC [64] LOG:  database system was shut down at 2025-02-14 09:02:50 UTC
2025-02-14 09:02:50.376 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 09:02:57.242 UTC [79] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 09:02:57.242 UTC [79] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 09:02:57.749 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:02:57.749 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_8b7bd767-d859-4304-a9f7-c3798278e5fe" WITH (FORCE)
2025-02-14 09:02:58.323 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:02:58.323 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_422ef773-761d-4038-bbc1-08ed0219a715" WITH (FORCE)
2025-02-14 09:02:58.747 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:02:58.747 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_4992a8f2-8c23-4075-bbf1-f2bd245453bb" WITH (FORCE)
2025-02-14 09:02:59.272 UTC [79] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:02:59.272 UTC [79] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_9078e8b5-232c-419e-893d-dfe1fe929953" WITH (FORCE)
